flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/converters/SqlShowFunctionsConverter.java

```diff
@@ -19,50 +19,49 @@
 package org.apache.flink.table.planner.operations.converters;
 
 import org.apache.flink.sql.parser.dql.SqlShowFunctions;
-import org.apache.flink.table.api.ValidationException;
-import org.apache.flink.table.catalog.CatalogManager;
 import org.apache.flink.table.operations.Operation;
 import org.apache.flink.table.operations.ShowFunctionsOperation;
+import org.apache.flink.table.operations.ShowFunctionsOperation.FunctionScope;
+import org.apache.flink.table.operations.utils.ShowLikeOperator;
 
 /** A converter for {@link SqlShowFunctions}. */
-public class SqlShowFunctionsConverter implements SqlNodeConverter<SqlShowFunctions> {
+public class SqlShowFunctionsConverter extends AbstractSqlShowConverter<SqlShowFunctions> {
+
+    @Override
+    public Operation getOperationWithoutPrep(
+            String qualifiedCatalogName,
```

Comment:
rename back to `catalogName` and `databaseName` because  `getOperation` also just calls them this way.